### ISSUE TITLE
Login Error Popup

### ISSUE THREAD

**Issue by krak3n** [thisissoon/FM-Frontend#47]
Only people with a thisissoon.com or thishe.re account can login, If a user connects with the FM application from an invalid domain the API will raise a `422`:

``` json
{
  "message": "Validation Error", 
  "errors": {
    "code": "You need be a member of SOON_ or This Here"
  }
}
```

Would be good if we had an error popup or something like that.

**Comment by eopa** [comment]
@jamesjwarren maybe we need a way to display the error messages on the error pages

**Comment by krak3n** [comment]
The snippet above is actually incorrect, it should be like this:

``` json
{
  "message": "Validation Error", 
  "errors": {
    "code": [
      "You need be a member of SOON_ or This Here"
    ]
  }
}
```

**Comment by jamesjwarren** [comment]
Hmm, we could pass the error message as a URL param from the responseError interceptor when we redirect to the error page?

**Comment by eopa** [comment]
another option would be to find a way of using the request interceptor only on requests in `$routeProvider.resolve` then handle in view requests like `mute` separately so we just show a pop up rather then redirect the user to 500 page

**Comment by jamesjwarren** [comment]
This was resolved with #50
